Header header

float64 y1r
float64 y2r
float64 y3r
float64 y1l
float64 y2l
float64 y3l

float64 dy1r
float64 dy2r
float64 dy3r
float64 dy1l
float64 dy2l
float64 dy3l

float64 s
float64 ds

float64 lLegMotorCurrentA
float64 lLegMotorCurrentB
float64 lLegMotorCurrentHip
float64 rLegMotorCurrentA
float64 rLegMotorCurrentB
float64 rLegMotorCurrentHip

uint32 count
